src/d3a/models/area/__init__.py

```diff
@@ -109,12 +111,15 @@ def activate(self, bc=None):
 
             if self.budget_keeper:
                 self.budget_keeper.activate()
+        if self.transfer_fee_pct is None:
+            self.transfer_fee_pct = self.config.iaa_fee
 
         # Cycle markets without triggering it's own event chain.
         self._cycle_markets(_trigger_event=False)
 
         if not self.strategy and self.parent is not None:
             self.log.warning("No strategy. Using inter area agent.")
+            self.log.warning(f"PERCENTAGE GRID FEE {self.transfer_fee_pct}%")
```

Comment:
Make sure to remove it after the PR, since it spams the logs

Reply:
Its only triggered at the start of simulation. Not sure if at least once in log could clutter it. Might not be super important but in future when there is separate grid fee for different area. It might come handy to analyse.

Reply:
The problem I have with this is that it simply displays a setting. If one wants to see the setting value, it is available throughout the usual config locations, that's why I do not find it useful to be on the log. Usually on the log, even in warning level, we display the output of some calculations and not config values.